client/src/main/scala/drt/client/components/ForecastFileUploadPage.scala

```diff
@@ -32,6 +32,16 @@ object ForecastFileUploadPage {
         <.input(^.`type` := "button", ^.value := "Upload", ^.onClick ==> onSubmit(portCode)))
     )
 
+  val uploadingInProgress: String => VdomTagOf[Div] = { message =>
+    <.div(
+      heading,
+      <.br(),
+      <.div(s"Upload status : $message"),
```

Comment:
I noticed after uploading a file locally it says ```Upload status : null```. Could we fix this so it gives a meaningful message?